git_wrapper/remote.py

```diff
@@ -52,10 +52,12 @@ def add(self, name, url):
 
         return ret_status
 
-    def fetch(self, remote="origin"):
+    def fetch(self, remote="origin", prune=False, prune_tags=False):
         """Refresh the specified remote
 
            :param str remote: Remote to fetch
+           :param bool prune: True if you want to prune
+           :param bool prune_tags: True if you want to prune tags
```

Comment:
Any thoughts about this part?

> then in the code, ignore the flag and don't pass it to remote.fetch() if prune_tags is provided on its own without prune. (Just log that we're ignoring it in that case.)

I think this could be 3 lines or so, if prune is False but prune_tags is True, set prune_tags to False and log at info level that prune-tags will be ignored. I'm wary about the "strange interactions" described in the man-page when it's used on its own, as the command might expand again in the future.